document/core/syntax/instrindex.rst

```diff
@@ -3,27 +3,27 @@
 Index of Instructions
 ---------------------
 
-=======================================================  =========================  ==========================================  ========================================  ===============================================================
-Instruction                                              Opcode                     Type                                        Validation                                Execution
-=======================================================  =========================  ==========================================  ========================================  ===============================================================
-:math:`\UNREACHABLE`                                     :math:`\hex{00}`           :math:`[t_1^\ast] \to [t_2^\ast]`           :ref:`validation <valid-unreachable>`     :ref:`execution <exec-unreachable>`
-:math:`\NOP`                                             :math:`\hex{01}`           :math:`[] \to []`                           :ref:`validation <valid-nop>`             :ref:`execution <exec-nop>`
-:math:`\BLOCK~[t^?]`                                     :math:`\hex{02}`           :math:`[] \to [t^\ast]`                     :ref:`validation <valid-block>`           :ref:`execution <exec-block>`
-:math:`\LOOP~[t^?]`                                      :math:`\hex{03}`           :math:`[] \to [t^\ast]`                     :ref:`validation <valid-loop>`            :ref:`execution <exec-loop>`
-:math:`\IF~[t^?]`                                        :math:`\hex{04}`           :math:`[] \to [t^\ast]`                     :ref:`validation <valid-if>`              :ref:`execution <exec-if>`
+=======================================================  =========================  ==========================================  =============================================  ===============================================================
```

Comment:
Ugh. Interestingly, GH markdown isn't as strict with the table layout. Something about ReST, I guess.